src/materials/nodes/Line2NodeMaterial.js

```diff
@@ -309,7 +310,7 @@ class Line2NodeMaterial extends NodeMaterial {
 					if ( useAlphaToCoverage && renderer.samples > 1 ) {
 
 						const dnorm = norm.fwidth();
-						alpha.assign( smoothstep( dnorm.negate().add( 0.5 ), dnorm.add( 0.5 ), norm ).oneMinus() );
+						alpha.assign( alpha.mul( smoothstep( dnorm.negate().add( 0.5 ), dnorm.add( 0.5 ), norm ).oneMinus() ) );
```

Comment:
Let's try `alpha.mulAssign()` instead of `alpha.assign( alpha.mul(`

Reply:
changed to mulAssign